custom-words.txt

```diff
@@ -926,6 +928,7 @@ LRO's
 Lucene
 Lumia
 LUNs
+machines's
```

Comment:
This is not grammatically correct. Depending on the actual context of use, it's either machine's (belonging to a single machine) or machines' (possessive of multiple machines).

Reply:
The word isn't used in our doc, it is used for retrieveBootDiagnosticsData API(line 3790) and reapply API(line 3565). I added the word in here, because the PR was failing a spell check was failing on this word

Reply:
This is line 3565:
        "description": "The operation to reapply a virtual machine's state.",

This is line: 3790:
        "description": "The operation to retrieve SAS URIs for a virtual machine's boot diagnostic logs.",

Both are using the word correctly. However, we are not using the word correctly and I believe it's this that's causing the issue:

Line 12107:           "description": " ...  **ImageDefault** - The virtual machines's default patching configuration is used. ..."
-- should be: The virtual machine's default patching configuration is used.

Please correct our description, remove from the dictionary and the spell check issue should go away.

Reply:
Oh I missed seeing the extra 's'